tickets.md & bots.md (#44)
ScootKit/scnx-docs: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/scnx/guilds /bots.md

---
sidebar_position: 1
---

> # Bots von SCNX

Reviewer: ```suggestion
# Bots auf SCNX
```

> Diese Dokumente beinhaltet anwendbare Hilfen für den [Custom Bot](../../custom-bot/intro) und den

Reviewer: ```suggestion
Diese Dokumente beinhaltet anwendbare Hilfen für den [Eigenen Bot](../../custom-bot/intro) und den
```

> Diese Dokumente beinhaltet anwendbare Hilfen für den [Eigenen Bot](../../custom-bot/intro) und den

Reviewer 2: ```suggestion
Diese Dokumente beinhalten anwendbare Hilfen für den [Eigenen Bot](../../custom-bot/intro) und den
```

 [Modmail Bot](../../modmail/intro) - wenn du diese Bots einstellen willst, befolge bitte den
 [mit dem eigenen  Bot loslegen](../../custom-bot) oder den [mit dem Modmail-Bot loslegen](../../modmail) Artikel.

## Fehlerbehebung {#troubleshooting}

Die Schritte, um Fehler zu beheben, sind bei den beiden Bots unterschiedlich. Bitte schaue dir den passenden Abschnitt an:

* [Eigener Bot](./../../custom-bot/troubleshooting)

> * Modmail-Bot

Reviewer 2: Verlinkung fehlt

Author: Wo?

Reviewer: This is intended, the original article has no link either, this can be resolved.

## Die Grundlagen {#basics}

Jeder Bot auf SCNX ist ein eigener Prozess, der auf einem echten Server ausgeführt wird. Du kannst den Status im 
Bot-Status-Panel von deinem [Modmail-](https://scnx.app/glink?page=modmail/manage)
oder [Eigenen-Bot-](https://scnx.app/glink?page=bot/manage) Dashboard einsehen.

Hier ist ein Beispiel von einem eigenen Bot-Status-Panel:

![](@site/docs/assets/scnx/guilds/bots/bot-status-example.png)

> Wie du sehen kannst, gibt es hier beriets einige Möglichkeiten. Hier sind die wichtigsten Informationen, die du darüber wissen musst:

Reviewer: ```suggestion
Wie du sehen kannst, gibt es hier bereits einige Möglichkeiten. Hier sind die wichtigsten Informationen über das Panel:
```

> * Der Tag und das Profilbild deines Bots (nur sichtbar wenn dein Bot online ist). Um das Profil deines Bots zu wechseln,

Reviewer 2: ```suggestion
* Der Tag und das Profilbild deines Bots (nur sichtbar, wenn dein Bot online ist). Um das Profil deines Bots zu verändern,
```

> * Der Tag und das Profilbild deines Bots (nur sichbar wenn dein Bot online ist). Um das Profil deines Bots zu wechseln,

Reviewer: ```suggestion
* Der Tag und das Profilbild deines Bots (nur sichtbar wenn dein Bot online ist). Um das Profil deines Bots zu wechseln,
```

>  klicke auf dieses oder befolge [diesem Artikel](#change-profile).

Reviewer 2: ```suggestion
 klicke auf dieses oder befolge [diesen Artikel](#change-profile).
```

> * "Konfiguration neu laden": Wenn du deine Konfiguration neu lädst, wird dein Bot alle Änderungen übernommen (auch Modul-Änderungen)

Reviewer: ```suggestion
* "Konfiguration neuladen": Wenn du deine Konfiguration neulädst, wird dein Bot alle ausstehenden Änderungen an Konfigurationsdatein übernehmen (auch Modul-Änderungen)
```

> * "Konfiguration neuladen": Wenn du deine Konfiguration neulädst, wird dein Bot alle ausstehenden Änderungen an Konfigurationsdatein übernehmen (auch Modul-Änderungen)

Reviewer 2: ```suggestion
* "Konfiguration neuladen": Wenn du deine Konfiguration neulädst, wird dein Bot alle ausstehenden Änderungen an Konfigurationsdateien übernehmen (auch Modul-Änderungen)
```

  ohne dass der Bot offline geht. Generell solltest du lieber die Konfiguration neu laden als den Bot neuzustarten.
* "Bot neustarten": Dies wird den Prozess deines Bots neustarten. Dies wird eventuell einige Minuten in Anspruch nehmen,

>   zusätzlich wird der Bot für eine kurze Zeit offline sein, in dieser Zeit wird der Bot nicht erreichbar sein.

Author: Is it not correct? I think that’s easier to understand

>   zusätzlich wird der Bot für eine kurze Zeit offline sein, in dieser Zeit wird der Bot nicht erreichbar sein.

Reviewer: No, a repetition is informal in this case. The sentence can be clearly understood without repeating "Zeit" another time. Wortwiederholungen are informal and should be avoided in texts with a professional audience. In this particular case, no argument for easier understanding can be made, as a clear us of the word "welcher" creates the connection to the last used object of the previous sentence.

 Du solltest deinen Bot nur neustarten, wenn du dazu aufgefordert wirst, seltsame Fehler auftreten oder du ein [Update durchführen willst](#bot-updates).

> * "Bot anhalten": Dies wird den Prozess deines Bots beenden. Es wird nicht auf Discord erreichbar sein. Es kann bis zu 5 Minuten dauern, bis der Bot in Discord als offline angezeigt wird.

Reviewer 2: ```suggestion
* "Bot anhalten": Dies wird den Prozess deines Bots beenden. Er wird nicht auf Discord erreichbar sein. Es kann bis zu 5 Minuten dauern, bis der Bot in Discord als offline angezeigt wird.
```

> * "Log" / "Fehler-Log": In diesen Log-Datein wird dein Bot dokumentieren, was er macht und wird die Gründe von Fehlern und Problemen aufschreiben. Unser Team (und technische Experten) benötigen diese Logs, um eventuelle Probleme zu analysieren und dir eine Lösung vorzuschlagen - oder, falls es Fehler im Bot selbst sind, diese zu reparieren. Eventuell wirst du gefragt, die [Logs zu teilen](#sharing-logs).

Reviewer 2: ```suggestion
* "Log" / "Fehler-Log": In diesen Log-Dateien wird dein Bot dokumentieren, was er macht und wird die Gründe von Fehlern und Problemen aufschreiben. Unser Team (und technische Experten) benötigen diese Logs, um eventuelle Probleme zu analysieren und dir eine Lösung vorzuschlagen - oder, falls es Fehler im Bot selbst sind, diese zu reparieren. Eventuell wirst du gefragt, die [Logs zu teilen](#sharing-logs).
```

* Sprache: Das ist die (menschliche) Sprache die dein Bot für alle Standardkonfigurationswerte und Nachrichten, die nicht konfiguriert werden können, nutzt. Der Modmail kann in manchen Fällen die Sprache des ausführenden Nutzer benutzen, statt der hier eingestellten Sprache. Du kannst jederzeit [die Sprache deines Bots ändern](#bot-language).

> * Server: Dies ist der physische Server - auch Bot-Host genannt - auf dem dein Bot betrieben wird. Du kannst den [Bot-Host wechseln](#bot-host) wenn du möchtest.

Reviewer 2: ```suggestion
* Server: Dies ist der physische Server - auch Bot-Host genannt - auf dem dein Bot betrieben wird. Du kannst den [Bot-Host wechseln](#bot-host), wenn du möchtest.
```

> * Branch (Nur bei Custom-Bots verfügbar): Der aktuelle Zweig (englisch "branch") ist eine Version deines eigenen Bots auf einer anderen Programmier-Basis. Du solltest es diesen bei "v3" (Stabie Version) belassen, aber wenn du willst, kannst du auch zu dem "Beta"-Branch wechseln (das ist die Öffentliche Beta; kann aber instabil sein).

Reviewer: ```suggestion
* Branch (Nur bei Custom-Bots verfügbar): Der aktuelle Zweig (englisch "branch") ist eine Version deines eigenen Bots auf einer anderen Programmier-Basis. Du solltest es diesen bei "v3" (Stabile Version) belassen, aber wenn du willst, kannst du auch zu dem "Beta"-Branch wechseln (das ist die Öffentliche Beta; kann aber instabil sein).
```

> * Branch (Nur bei Custom-Bots verfügbar): Der aktuelle Zweig (englisch "branch") ist eine Version deines eigenen Bots auf einer anderen Programmier-Basis. Du solltest es diesen bei "v3" (Stabile Version) belassen, aber wenn du willst, kannst du auch zu dem "Beta"-Branch wechseln (das ist die Öffentliche Beta; kann aber instabil sein).

Reviewer 2: ```suggestion
* Branch (nur beim eigenen Bot verfügbar): Der aktuelle Zweig (englisch "branch") ist eine Version deines eigenen Bots auf einer anderen Programmier-Basis. Du solltest es diesen bei "v3" (Stabile Version) belassen, aber wenn du willst, kannst du auch zu dem "Beta"-Branch wechseln (das ist die öffentliche Beta; kann aber instabil sein).
```

> * Version (Nur bei Custom-Bots sichtbar): Dies ist die Anzahl an veröffentlichen code-versionen. Abhängig von deiner Version wird dein Bot eventuell keine neuen Funktionen nutzen können. Wenn deine Version nicht aktuell ist wird es eine Warnung geben, die dies zeigt. Dein Modmail Bot wird immer auf dem neusten Stand sein. Erfahre mehr über [Updates & Versionen](#bot-updates).

Author: like you said, its "eigener bot", not "custom bot". can you change it? (I cant or Im to dumb)

> * Version (Nur bei Custom-Bots sichtbar): Dies ist die Anzahl an veröffentlichen code-versionen. Abhängig von deiner Version wird dein Bot eventuell keine neuen Funktionen nutzen können. Wenn deine Version nicht aktuell ist wird es eine Warnung geben, die dies zeigt. Dein Modmail Bot wird immer auf dem neusten Stand sein. Erfahre mehr über [Updates & Versionen](#bot-updates).

Reviewer: :+1:

> * Version (Nur bei eigenen Bots sichtbar): Dies ist die aktuelle Version des Codes deines Bots. Abhängig von deiner Version wird dein Bot eventuell neuen Funktionen nicht nutzen können. Wenn deine Version nicht aktuell ist, wird eine Warnung angezeigt. Dein Modmail Bot wird immer automatisch auf dem neusten Stand aktualisiert. Erfahre mehr über [Updates & Versionen](#bot-updates).

Reviewer 2: ```suggestion
* Version (nur bei eigenen Bots sichtbar): Dies ist die aktuelle Version des Codes deines Bots. Abhängig von deiner Version wird dein Bot eventuell neue Funktionen nicht nutzen können. Wenn deine Version nicht aktuell ist, wird eine Warnung angezeigt. Dein Modmail-Bot wird immer automatisch auf den neusten Stand aktualisiert. Erfahre mehr über [Updates & Versionen](#bot-updates).
```

* RAM / CPU: Dies wird direkt unter dem "Online"-Status angezeigt. Du kannst sie ohne Sorgen ignorieren, sie zeigen nur die RAM / CPU Nutzung deines Bots an. Diese sind irrelevant, da SCNX automatisch deinen Bot optimiert, falls dein Bot zu viel RAM friss[t](https://www.linuxatemyram.com/) als er tun sollte.

### Bot-Updates {#bot-updates}

<Tabs groupId="scnx-bot-type-k">

>     <TabItem value="customBot" label="Custom Bot">

Reviewer: ```suggestion
    <TabItem value="customBot" label="Eigener Bot">
```

        <ul>

>             <li>Wir veröffentlichen regelmäßig Updates für Discord-Bots - wir stellen meist mehrere Funktionen in ein Update für die öffentliche Version. Für die <a href="#beta">Öffentliche Beta</a> gibt es viele Veröffentlichungen, manchmal mehrere an einem Tag.</li>

Reviewer: ```suggestion
            <li>Wir veröffentlichen regelmäßig Updates für Discord-Bots - wir fassen meist mehrere Funktionen in einem Update für die stabile Version zusammen. Für die <a href="#beta">Öffentliche Beta</a> gibt es viele Veröffentlichungen mit weniger Änderungen pro Veröffentlichung, manchmal mehrere an einem Tag.</li>
```

            <li>Dein Bot wird eine Nachricht in deinen Bot-Log-Channel (falls eingestellt) senden, wenn ein neues Update verfügbar ist.</li>

>             <li>Wir kündingen jedes Update auf unserem <a href="https://scootk.it/dc">Discord-Server</a> an und senden einen Link zu dem Changelog.</li>

Reviewer: In German, you can't use "dem Changelog" without first establishing which object you are referring to.

            <li>Um die neuste Version des Bots zu nutzen, starte den Bot ganz einfach in deinem <a href="https://scnx.app/de/glink?page=bot/manage">Bot-Dashboard</a> neu. Updates Updates sind immer kostenlos.</li>
            <li>Du wirst keine neuen Funktionen nutzen können, wenn dein Bot nicht auf der aktuellsten Version ist, bis du deinen Bot auf die neuste Version bringst.</li>
        </ul>
    </TabItem>
    <TabItem value="modmailBot" label="Modmail-Bot">
        <ul>
          <li>Wir veröffentlichen regelmäßig Updates für den Modmail Bot, manchmal mehrere an einem Tag. Alle Updates sind kostenlos.</li>
          <li>Dein Modmail Bot wird sich immer auf der neusten ihm verfügbaren Version befinden.</li>
          <li>Wenn ein neues Update für deinen Modmail Bot verfügbar ist, wird dein Bot das Update automatisch installieren.</li>
          <li>Wir kündigen größere Updates auf unserem <a href="https://scootk.it/dc">Discord-Server</a> an und posten einen Link zu dem Änderungsbericht.</li>

>           <li>Manche Updates oder Fehler-behebungen werden nicht angekündigt - dein Modmail Bot wird sie einfach istallieren.</li>

Reviewer: ```suggestion
          <li>Manche Updates oder Fehlerbehebungen werden nicht angekündigt - dein Modmail Bot wird sie einfach installieren.</li>
```

          <li>Du findest alle Änderungsberichte in deinem <a href="https://scnx.app/de/glink?page=modmail/manage">Bot-Dashboard</a> und auf unserer <a href="https://scnx.app/de/changelogs?type=MODMAIL">Änderungsberichte Seite</a>.</li>
          <li>Wenn dein Bot ein Update durchführt, wird er für eine kurze Zeit offline sein, während dein Bot einige Datenbanken-Änderungen durchführt. Dies wird normalerweise unter einer Minute dauern und dein Bot arbeitet danach wieder wie davor. </li>
        </ul>
    </TabItem>
</Tabs>

### Teilnahme an der öffentlichen Beta {#beta}

<Tabs groupId="scnx-bot-type-k">
    <TabItem value="customBot" label="Eigener Bot">
        Bevor du die öffentliche Beta verwendest, beachte bitte:
        <ul>

>           <li>Die öffentliche Beta kann instabil sein. Das bedeutet, dass es passieren kann, dass dein Bot crashed oder neue Funktionen nicht korrekt funktionieren.</li>

Reviewer: ```suggestion
          <li>Die öffentliche Beta kann instabil sein. Das bedeutet, dass es passieren kann, dass dein Bot abstürzt oder neue Funktionen nicht korrekt funktionieren.</li>
```

          <li>Manche neue Funktionen in der Beta sind noch nicht übersetzt. Das bedeutet, dass dein Bot eventuell Englisch spricht, anstatt deine voreingestellte Sprache.</li>
          <li>Da du die öffentliche Beta jederzeit verlassen kannst, kann es sein, dass dein Bot kaputt geht wenn die Beta-Version eine höhere Veröffentlichungsnummer als die ältere Version hat. Darüber hinaus könnten Konfigurationsdateien nach dem Zurückschalten beschädigt sein und müssen zurückgesetzt werden, was zu Datenverlust führt.</li>
        </ul>
        So kannst du die Custom-Bot Beta verwenden:
        <ol>
          <li>Öffne das <a href="https://scnx.app/glink?page=bot/manage">Bot-Dashboard</a>.</li>
          <li>In dem Status-Panel, klicke auf "Wechseln" dann auf "Branch"-Einstellungen.</li>
          <li>Wähle "Öffentliche Beta (kann instabil sein)"aus.</li>
          <li>Bestätige die Änderung, indem du auf "Speichern" klickst.</li>

>           <li>Um Update Benachrichtigungen über Beta Veröffentlichungen auf unserem <a href="https://scootk.it/dc">Discord</a> zu erhalten, aktiviere die Beat Kanäle in #beta.</li>

Author: me and typing...

>           <li>Um Benachrichtigungen über die Veröffentlichung von neuen Beta-Version auf unserem <href="https://scootk.it/dc">Discord</a> zu erhalten, aktiviere die Beta-Kanäle in #beta.</li>

Reviewer: ```suggestion
          <li>Um Benachrichtigungen über die Veröffentlichung von neuen Beta-Version auf unserem <a href="https://scootk.it/dc">Discord</a> zu erhalten, aktiviere die Beta-Kanäle in #beta.</li>
```

broken html

        <ol>
          <li>Öffne das <a href="https://scnx.app/glink?page=bot/manage">Bot-Dashboard</a>.</li>
          <li>In dem Status-Panel, klicke auf "Change" dann auf "Branch"-Einstellung.</li>
          <li>Wähle "Stabile Version (empfohlen)" aus.</li>
          <li>Bestätige die Änderung, indem du auf "Speichern" klickst.</li>
        </ol>
    </TabItem>
    <TabItem value="modmailBot" label="Modmail-Bot">Momentan gibt es keine öffentliche Beta Version für unseren Modmail Bot. Ausgewählte Nutzer erhalten Zugriff auf die Private Beta, aber sonst werden alle neuen Funktionen für alle Nutzer verfügbar sein. Im Gegensatz zu unserem Custom-Bot testen wir die Modmail Funktionen ausführlich und stellen sicher, dass jede neue Version komplett sicher ist. </TabItem>
</Tabs>

### Den Bot-Host wechseln {#bot-host}

Der Bot-Host ist der physikale Server auf dem dein Bot läuft. Wir bieten viele Bot-Hosts an, die rund um die Welt verteilt sind. So kannst du den Bot-Host wechseln:
1. Öffne das [Modmail-](https://scnx.app/glink?page=modmail/manage)
   oder das [CustomBot-](https://scnx.app/glink?page=bot/manage) Dashboard.
2. Als nächstes klicke auf "wechseln" in dem "Status" Bereich.

> 3. Wähle den Bot-Host aus, auf den du welchseln willst.

Author: i checked it with lanuage tool, it also says that it is "wechseln"

> 3. Wähle den Bot-Host aus, auf den du Welchseln willst.

Reviewer: ```suggestion
3. Wähle den Bot-Host aus, auf den du Wechseln willst.
```

4. Klicke auf "Bestätigen".

<details>
  <summary>Was ist der Unterschied zwischen dem "Premium" und den anderen Bot-Hosts?</summary>
  <ul>
    <li>Die meisten Bot-Hosts liegen in Nord Amerika und haben eine schnellere Verbindung zu der Infrastruktur von Discord.</li>
    <li> Die Infrastruktur von Premium-Hosts haben höhere Zuverlässigkeit, höhere Bandbreite und mehr.</li>

>     <li>Jedem Bot werden mehr Ressourcen zugewiesen.</li>

Author: it sounds not so perfect, if you ask me it should be "Jedem Bot , der sich auf einem Premium Bot-Hosts befindet, werden mehr Ressourcen zugewiesen."

>     <li>Jedem Bot werden mehr Ressourcen zugewiesen.</li>

Reviewer: lots of words to say the same things, the original string was still wrong as it changed the meaning of the translation

    <li>Premium Bot-Hosts können doppelt so viele Backups speichern und lagern sie wo anders als die anderen Bot-Hosts.</li>
  </ul>
</details>

<details>
  <summary>Welcher Bot-Host ist der beste? / Welchen Bot-Host sollte ich nehmen?</summary>

Letzten Endes ist es egal (bis auf die Premium-Server). Jeder Bot-Host ist dafür da, Bots wie deinen, zu hosten.

> Wir haben hohe Standarts bei den Bot-Hosts und können versichern, dass alle das gleiche Erlebniss geben. Eventuell werden wir deinen Bot zwischen den Bot-Hosts verschieben, wenn du keinen bezahlten Plan hast, um Ressourcen zu sparen.

Reviewer: ```suggestion
Wir haben hohe Standards bei den Bot-Hosts und können versichern, dass alle das gleiche Erlebnis bieten. Eventuell werden wir deinen Bot zwischen den Bot-Hosts verschieben, wenn du keinen bezahlten Plan hast, um Ressourcen zu sparen.
```

</details>

<details>
    <summary>Ich wurde gebeten, bestimmte europäische Datenschutzrechte aufzugeben, wenn ich meinen Bot außerhalb der EU verlagere</summary>

    Wenn du die Daten deines Bots außerhalb der Europäischen Union verschiebst, gilt die Datenschutz-Grundverordnung nicht mehr für deine Daten. Dies betrifft nur die Daten deines Bots, wie Leveling-Stats, Konfiguration und das eigentliche Hosting deines Bots. Bitte bedenke auch, dass Discord selbst
     die meisten deiner Daten auch außerhalb der EU speichert und nur mit vertrauenswürdigen Hosting-Anbietern mit hohen Sicherheits- und Datenschutzstandards zusammen arbeitet; Es gelten die Datenschutzgesetze des jeweiligen Landes. Dies bedeutet manchmal, dass wir deine Daten nicht vor der Nutzung/Einsichtnahme durch staatliche Stellen schützen können.
    Da es gesetzlich vorgeschrieben ist (zumindest für unsere EU-Kunden ♥), müssen Sie einer solchen Datenübermittlung zustimmen. Du kannst den Hosting-Standort deines Bots jederzeit wieder auf Bot-Host innerhalb der EU umstellen.

</details>

### Die Bot-Sprache ändern {#bot-language}

> Die (Menschliche) Sprache wird für alle Standart-konfiguration-eingaben und Nachrichten die nicht eingestellt werden können verwendet. Es kann vorkommen, dass der Modmail Bot nicht diese, sondern die Sprache, die der ausführende Nutzer eingestellt hat, nutzt.

Author: Standardtkonfigurationwete :/
Standardtkonfigurationwerte

> Die (Menschliche) Sprache wird für alle Standart-konfiguration-eingaben und Nachrichten die nicht eingestellt werden können verwendet. Es kann vorkommen, dass der Modmail Bot nicht diese, sondern die Sprache, die der ausführende Nutzer eingestellt hat, nutzt.

Reviewer: :+1:

> Die (Menschliche) Sprache wird für alle Standart-konfiguration-eingaben und Nachrichten die nicht eingestellt werden können verwendet. Es kann vorkommen, dass der Modmail Bot nicht diese, sondern die Sprache, die der ausführende Nutzer eingestellt hat, nutzt.

Author: some things are optional to change...
and e.g. adding in links "de/" is something that i didnt know

:::info

> Wenn du die Bot Sprache wechselst, bitte beachte dass du nicht die eingestellten Datein neu generierst - du müsstest die Datei zurücksetzen oder es selbst übersetzen.

Author: bitt -> bitte

:::

Um die Sprache deines Bots zu ändern,

1. Öffne das [Modmail-](https://scnx.app/glink?page=modmail/manage)
   oder das [CustomBot-](https://scnx.app/glink?page=bot/manage) Dashboard.

> 2. Als nächtes klicke auf "Wechseln", dann auf "Server" in der "Status"-Abteilung.

Author: huh? i dont think so

> 2. Klicke als Nächtes auf "Wechseln", dann auf "Server" in der "Status"-Abteilung.

Reviewer: ```suggestion
2. Klicke unter "Bot-Status" bei der Sprache deines Bots auf "Wechseln".
```

</details>

## Erwähnung von SCNX entfernen {#offbrand}

<IncludedInPlan data={{PRO: true, ACTIVE_GUILD: false, STARTER: false}} />

Du kannst immer den Footer von allen Nachrichten ändern von allen Nachrichten die du auf SCNX bearbeiten kannst, aber du kannst auch die voreingestellte Nachricht ändern wo du den Footer nicht manuell ändern kannst oder es dafür kein vorgesehenes Feld gibt. 

Um das "Powered by scnx.xyz ⚡" von *jedem* Embed von deinem Bot zu entfernen, folge bitte diesen Schritten:

<Tabs groupId="scnx-bot-type-k">
    <TabItem value="customBot" label="Custom Bot">
        <ol>
            <li>Öffne die <a href="https://scnx.app/glink?page=bot/configuration?file=bot%7Cstrings">Nachrichten & Aussehen</a> Einstellung von der Eingebauten-konfiguration deines Bots.</li>
            <li>Verändere den "Embed-Footer" und die "Embed-Footer-Image-URL" wie du möchtest. Du kannst eine Bild-URL in deinem<a href="https://scnx.app/user/files">Datein Speicher</a> hochladen - lade einfach ein Bild dort hoch und klicke auf "Öffentliche URL kopieren".</li>

>             <li>Speichere die Änderungen & starte den Bot neu um die Änderungen anzunehmen.</li>

Author: :/

>             <li>Speichere die Änderungen & starte den Bot neu um die Änderungen anzunehmen.</li>

Reviewer: (:

>             <li>Speichere die Änderungen & starte den Bot neu um die Änderungen anzunehmen.</li>

Author: optional

        </ol>
    </TabItem>
     <TabItem value="modmailBot" label="Modmail-Bot">
        <ol>
            <li>Öffne die <a href="https://scnx.app/glink?page=modmail/configuration?show=branding">Bot Branding</a> Abteilung von deiner Modmail Konfiguration.</li>
            <li>Ändere den "Standart-Embed-Fußzeile" und das "Standart-Fußzeilen-Bild" wie du möchtest. Du kannst ganz einfach ein Bild hinzufügen oder entfernen, falls du es nicht mehr brauchst.</li>

>             <li>Sichere die Änderungen & lade die Konfiguration neu um die Änderungen abzunehmen. </li>

Author: typo, i wanted do write anzunehmen

>             <li>Sichere die Änderungen & lade die Konfiguration neu um die Änderungen abzunehmen. </li>

Reviewer: anzunehmen is not fitting in this context.

        </ol>
    </TabItem>
</Tabs>

## Namen, das Über mich oder Profilbild meines Bot ändern {#change-profile}

:::info Bevor du weiterliest:

> * damit dies funktioniert muss dein Bot auf SCNX online sein. Wenn dein Bot nicht auf SCNX online sein sollte, schaue bitte in die [Anleitung zur Fehlerbehebung](#troubleshooting).

Reviewer: ```suggestion
* Damit dies funktioniert muss dein Bot auf SCNX online sein. Wenn dein Bot nicht auf SCNX online sein sollte, schaue bitte in die [Anleitung zur Fehlerbehebung](#troubleshooting).
```

* Wenn du nicht der Inhaber von dem Server bist, musst du die [Trusted-Admin Berechtigung](./trusted-admins#permissions) "Bot verwalten" auf dem Produkt haben, dessen Einstellungen du ändern möchtest.
:::

So kannst du das Profil-Bild und den Nutzernamen deines Bots ändern, ohne SCNX zu verlassen:

> 1.Öffne das [Modmail-](https://scnx.app/de/glink?page=modmail/manage?showEditProfile=true)

Reviewer: ```suggestion
1. Öffne das [Modmail-](https://scnx.app/de/glink?page=modmail/manage?showEditProfile=true)
```

    <ul>
        <li>Bitte stelle sicher, dass dein Bot online ist und ohne Fehler läuft. Wenn dies nicht der Fall sein sollte, schaue dir bitte unsere <a href="#troubleshooting">Anleitung zur Fehlerbehebung</a> an.</li>
        <li>Manche Namen sind reserviert oder wurden bereits zu oft genutzt und können nicht für deinen Bot verwendet werden. Versuche es bitte mit einem anderen, weniger beliebten, Namen erneut.</li>
        <li>Wenn du ein Profilbild für deinen Bot hochlädst, verbraucht dies deinen Nutzerspeicher. Bitte öffne das <a href="https://scnx.app/user/files">Datein-Dashboard</a> und schaue nach ob genug Platz für dein neues Profil-Bild vorhanden ist.</li>
        <li>Du (oder dein Bot) könnten rate-limitiert sein. Bitte versuche es in 1-4 Stunden erneut.</li>
        <li>Du wirst den genauen Grund in dem Log deines Bots finden. Bitte <a href="#sharing-logs">teile deine Logs</a> und versuche den Fehler darin zu finden. Wenn du ihn nicht findest, <a href="https://scnx.app/help">kontaktiere bitte unser Team</a>.</li>
    </ul>
</details>

## Meinen Bot einladen {#invite-bot}

:::info Bevor du weiterliest

> * Dafür muss dein Bot online sein. Wenn dein Bot nicht auf SCNX online ist, öffne bitte den Abschnitt „Der Bot ist nicht auf meinem Server“ im unserem [Leitfaden zur Fehlerbehebung](#troubleshooting).

Reviewer: ```suggestion
Dafür muss dein Bot online sein. Wenn dein Bot nicht auf SCNX online ist, öffne bitte den Abschnitt „Der Bot ist nicht auf meinem Server“ im unserem [Leitfaden zur Fehlerbehebung](#troubleshooting).
```

:::

1. Öffne das [Modmail-](https://scnx.app/glink?page=modmail/manage)
   oder das [eigene Bot-](https://scnx.app/glink?page=bot/manage) Dashboard.
2. Klicke anschließend in der Bot-Statuskarte auf „Erweiterte Optionen“.
3. Klicke auf "Invite-URL anzeigen" und klicke dann auf "Invite-URL kopieren".
4. Klicke auf "Abbrechen".

## Logs teilen {#sharing-logs}

Bevor wir loslegen, eine kurze Erklärung darüber, was Logs überhaupt sind und warum du sie teilen solltest. Jeder Bot hat seinen eigenen Prozess auf SCNX, der auf einem echten, physischen Server verwaltet wird. Während dein Bot läuft, produziert dieser Prozess solche Log-Datein. In diesen Datein dokumentiert dein Bot, alles, was was er macht und schreibt den Grund für Fehlern auf. Unser Team benötigt diese Logs. um eventuelle Probleme zu diagnostizieren und dir eine Lösung anzubieten - oder um, wenn es Fehler direkt am Bot, ist - diese zu beheben.

:::info

> Bot Logs zu Teilen, ist eine „Einmalige Erfassung“ davon - geteilte Logs werden nicht aktualisiert, wenn dein Bot etwas Neues dokumentiert. Die geteilten Logs werden nach 7 Tagen,  nachdem du sie geteilt hast, gelöscht. Logs beinhalten meinst keine privaten Informationen, aber bitte schaue trotzdem drüber, bevor du sie veröffentlichst.

Reviewer: ```suggestion
Bot Logs zu Teilen, ist eine „Einmalige Erfassung“ davon - geteilte Logs werden nicht aktualisiert, wenn dein Bot etwas Neues dokumentiert. Die geteilten Logs werden nach 7 Tagen,  nachdem du sie geteilt hast, gelöscht. Logs beinhalten meinst keine privaten Informationen, aber bitte schaue trotzdem darüber, bevor du sie veröffentlichst.
```

:::

:::caution
Bitte verschwende nicht die Zeit unseres Teams, falls du einen Fehler in den Logs findest. Es ist normal, dass dein Bot manchmal Fehler dokumentiert. Wir erhalten diese automatisch, und werden diese beheben. Teile Logs bitte nur, wenn du wirklich Probleme am Bot feststellst.
:::

So kannst du die Bot Logs teilen:

1. Öffne das [Modmail-](https://scnx.app/de/glink?page=modmail/manage)
   oder das [eigene Bot-](https://scnx.app/de/glink?page=bot/manage) Dashboard.

> 2. Als Nächstes klicke auf "mehr Optionen" in deinem Bot-Status-Panel.

Reviewer: ```suggestion
2. Als Nächstes klicke auf "Mehr Optionen" in deinem Bot-Status-Panel.
```

![](@site/docs/assets/scnx/guilds/bots/change-token.png)

Falls dein Bot danach nicht mehr Funktionieren sollte, befolge bitte die [Anleitung zur Fehlerbehebung](#troubleshooting).

## Bot Löschen {#delete-bot}

:::danger

Wenn du deinen Bot löschst, wird dies nicht nur für immer die Bot Konfiguration löschen, sondern wird auch alle Daten löschen (wie Level von Nutzern oder Ticket-Daten).

:::

:::caution

* Du musst der Inhaber des Servers sein, um den Bot zu löschen.
* Löschen des Bots wird höchstwahrscheinlich **keine Probleme lösen**. Bitte nutze die [Anleitung zur Fehlerbehebung](#troubleshooting), um Fehler zu diagnostizieren und zu beheben.
* Wenn du den Nutzernamen oder das Profilbild deines Bots ändern willst, folge [dieser Anleitung](#change-profile).

:::

Es tut uns leid, dass du SCNX verlassen willst - wenn du Probleme mit SCNX hast, [helfen wir dir gerne](https://scnx.app/de/help) auf unserem [Discord](https://sc-net.work/dc).

Um deinen Bot  (für immer) zu löschen, folge dieser Anleitung:

1. Öffne das [Modmail-](https://scnx.app/de/glink?page=modmail/manage)
   oder das [eigene Bot-](https://scnx.app/de/glink?page=bot/manage) Dashboard.

> 2. Als Nächstes, klicke auf "mehr Optionen" in dem Bot-Status-Panel.

Reviewer: ```suggestion
2. Als Nächstes, klicke auf "Mehr Optionen" in dem Bot-Status-Panel.
```
